EnglishFilter/sections/foreign.txt

```diff
@@ -4753,6 +4753,9 @@ fxp.co.il#@#iframe[width="300"][height="250"]
 !------------------------------ Italian ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74605
+@@||privacy-center.org/*/loader.js$domain=tuttosport.com
+@@||privacy-center.org/sdk.*.js$domain=tuttosport.com
```

Comment:
Wrong filter. Look at the @AdamWr [post](https://github.com/AdguardTeam/AdguardFilters/pull/74987#issuecomment-779335609).